introspection-engine/introspection-engine-tests/tests/relations/postgres.rs

```diff
@@ -71,7 +71,7 @@ async fn multiple_foreign_key_constraints_are_taken_always_in_the_some_order_coc
         model A {
           id  Int @id
           foo Int
-          B   B   @relation(fields: [foo], references: [id], onDelete: Cascade, map: "fk_1")
+          B   B   @relation(fields: [foo], references: [id], onDelete: Cascade, onUpdate: NoAction, map: "fk_1")
```

Comment:
locally, it passes with and without this field. on CI, it needs this field. i'm not sure, but it is still correct either way...

Reply:
it may also because i'm getting a different random seed on CI, and it passed by chance because it always hits the "else". in CI it complains if the answer is fk_1.